internal/guest/bridge/bridge.go

```diff
@@ -384,7 +389,7 @@ func (b *Bridge) ListenAndServe(bridgeIn io.ReadCloser, bridgeOut io.WriteCloser
 
 			s := trace.FromContext(resp.ctx)
```

Comment:
It looks like there might already be a span created when receiving a response, so it might be better to just rely on that

Reply:
but looks like the span could be nil too so that's why I thought of starting a span explicitly for each go func()